### DISCUSSION TITLE
Project durability

### DISCUSSION THREAD

**Discussion by pguiomar** [getgauge/taiko#2707] [General]
Hi.
Further to this post [https://github.com/getgauge/gauge/issues/1732](url),
are the GAUGE + TAÏKO projects still active and maintenance assured ?

**Comment by zabil** [comment]
Gauge and Taiko is not actively sponsored or maintained. 

However, pull requests for bug fixes are welcome.

**Reply by maxshifrin** [reply]
To follow up on the question above, is it possible to get paid support and maintenance for these?

**Reply by zabil** [reply]
I am not aware if anyone's offering paid support for Gauge and Taiko at the moment. So will leave this question open.

**Comment by pguiomar** [comment]
Thanks for this reply.

**Comment by martasus** [comment]
thanks
